### ISSUE TITLE
Expose val/loss during model training

### ISSUE THREAD

**Issue by j99ca** [open-edge-platform/geti#7413]
Getitune doesn't seem to compute/expose val/loss during training. I would like to set the adaptive_early_stopping and modelcheckpoint monitor in my multiclass classification training to minimize on val/loss (rather than maximizing on val/f1). If this is already possible, I would like to know how to do this.

Thanks!

**Comment by kprokofi** [comment]
Hi @j99ca,
Indeed, we don't compute loss value during validation for now and therefore it cannot be set for early stopping.
We do have possibility to change target metric in our recipes via callback_monitor parameter:

<img width="953" height="226" alt="Image" src="https://github.com/user-attachments/assets/ca49778b-0b3f-474f-9f9b-3f957d1825b2" />

We will expose in application configurable target validation metrics in the next releases and include validation loss computation as well.
Thank you for this proposal.
